Classes/Issues/IssuesViewController.swift

```diff
@@ -36,7 +36,7 @@ final class IssuesViewController:
     MessageTextViewListener {
 
     private let client: GithubClient
-    private let model: IssueDetailsModel
+    let model: IssueDetailsModel
```

Comment:
Why did you change the access control? If there is a reason, move it before the `private` properties 😊

Reply:
I did this so that I could reach the property from the `NotificationSectionController`

Reply:
Have moved